lib/timers.js

```diff
@@ -481,6 +481,7 @@ Timeout.prototype.unref = function() {
     }
 
     var handle = reuse(this);
+    delete handle._list;
```

Comment:
We could, I thought those were exposable somehow too?

I still think we should unreference it in some way though so the list can be GC'd.